src/rtc/core.py

```diff
@@ -1,7 +1,11 @@
 #!/usr/bin/env python
 
+import os
 import sys
+import shutil
 import logging
+import tempfile
```

Comment:
nit - sort the import order above

Reply:
I usually import the most generic library first, which sometimes can be subjective. Stardard libraries before third-party libraries and both before local libraries. Do you have any other suggestions? You are probably thinking of using the alphabetical order?

Reply:
Yes, I meant the alphabetical order for the standard libraries, but let's keep the order as it is as we discussed.